src/apps/intel_mp/intel_mp.lua

```diff
@@ -375,6 +429,15 @@ function Intel:init_rx_q ()
    local rxdesc_ring_t = ffi.typeof("$[$]", rxdesc_t, self.ndesc)
    self.rxdesc = ffi.cast(ffi.typeof("$&", rxdesc_ring_t),
    memory.dma_alloc(ffi.sizeof(rxdesc_ring_t)))
+
+   -- VMDq pool state (4.6.10.1.4)
+   if self.vmdq then
+      -- packet splitting none, enable 4 RSS queues per pool
+      self.r.PSRTYPE[self.poolnum](bits { RQPL=30 })
+      -- multicast promiscuous, broadcast accept, accept untagged pkts
+      self.r.PFVML2FLT[self.poolnum]:set(bits { MPE=28, BAM=27, AUPE=24 })
```

Comment:
Do untagged incoming packets get copied to all pools? Are all pools able to send untagged packets?